test/system/projects/samples_test.rb

```diff
@@ -261,6 +261,8 @@ class SamplesTest < ApplicationSystemTestCase
       end
       click_link I18n.t('projects.samples.show.concatenate_button'), match: :first
       within('span[data-controller-connected="true"] dialog') do
```

Comment:
Can I also get you to add a test for what the modal looks like when paired end files are selected to be concatenated

Reply:
fe4519bbbe9f6ad2777497a445be9da9dd315c7f